sql-statements/sql-statement-flush-tables.md

```diff
@@ -51,7 +51,7 @@ ERROR 1105 (HY000): FLUSH TABLES WITH READ LOCK is not supported.  Please use @@
 
 ## MySQL compatibility
 
-* TiDB does not have a concept of table cache as in MySQL. Thus, `FLUSH TABLES` is parsed but ignored in TiDB for compatibility.
+* TiDB does not have a concept of cached tables as in MySQL. Thus, `FLUSH TABLES` is parsed but ignored in TiDB for compatibility.
```

Comment:
This edit is incorrect. It refers to the MySQL table cache feature correctly (not to be confused with cached tables).

Reply:
```suggestion
* TiDB does not have a concept of table cache as in MySQL. Thus, `FLUSH TABLES` is parsed but ignored in TiDB for compatibility.
```